index.js

```diff
@@ -33,9 +49,22 @@ function list (cb) {
       })
       .map(function (line) {
         var name = line.slice(0, ssid).trim()
+        var bss = line.slice(ssid, bssid).trim()
+        var sig = line.slice(bssid, rssi).trim()
+        var chan = line.slice(rssi, channel).trim()
+        var htMode = line.slice(channel, ht).trim()
+        var countryCode = line.slice(ht, cc).trim()
         var sec = parseSecurity(line.slice(security).trim())
 
-        return {name: name, security: sec}
+        return {
+          name: name,
+          bssid: bss,
+          channel: ~~chan,
```

Comment:
It's a bitwise operator, which is used as a substitute for `Math.floor`. I might change this to `parseFloat`, but I don't think it would have any side effects in this case, as the numbers always are whole.